tests/test_serialization.py

```diff
@@ -18,6 +18,7 @@
     UnsignedShort: [0, 400],
     UnsignedLong: [0, 400],
     Integer: [-1000, 1000],
+    FixedPointInteger: [-1000, 1000],
```

Comment:
The values here should be `float`s, with at least one of them having a fractional part, as this is the actual use case of the FixedPointInteger type. (These test cases would not have caught the error from my previous comment).

Reply:
Oops my bad here.
Still getting the hang of these sort of things. Have amended this and added an additional float to check as well.
Thanks for the help and glad to help this project out! :)

Reply:
Sorry for being slow here - in the process of learning.
Values are now floats! However the test is failing
```

Reply:
Since this is a fix point with 5 bits dedicated to the decimal, it is more imprecise than floats and doubles. It should have its own case with that 3 set to a 2.